src/python/pcmdi/meanabs_xy.py

```diff
@@ -1,5 +1,5 @@
 import MV2 as MV
```

Comment:
@gleckler1 this import is inconsistent with the `src/python/pcmdi/bias.py` above - `import MV2` no `as MV`